lib/xml/kit/templatable.rb

```diff
@@ -88,7 +88,7 @@ def render(model, options)
       def signature_for(reference_id:, xml:)
         return unless sign?
 
-        signatures.build(reference_id).to_xml(xml: xml)
+        signatures.build([reference_id]).to_xml(xml: xml)
```

Comment:
Instead of requiring clients to pass an array can we accept either a single id or array via...

```ruby
reference_ids = Array(reference_ids)
```